index.html

```diff
@@ -36,7 +36,9 @@ <h5 class="card-title">Main advantages</h5>
         <h5 class="card-title">Roadmap</h5>
         <ul>
           <li>June 2019 - release <a href="https://github.com/orgs/opendevstack/projects/4">1.1</a></li>
-          <li>October 2019 - release <a href="https://github.com/orgs/opendevstack/projects/6">2.0</a></li>
+          <li>October 2019 - release <a href="https://github.com/orgs/opendevstack/projects/8">1.2</a></li>
+          <li>December 2019 - release <a href="https://github.com/orgs/opendevstack/projects/6">2.0</a></li>
+          <li>March 2020 - release <a href="https://github.com/orgs/opendevstack/projects/9">3.0</a></li>
```

Comment:
@tbugfinder as far as I know, 3.0 will come in June :)

Reply:
ok, the GitHub project has set: `Release date: May 2020`.
Is there a version / release tag expected for end-of-march?

Reply:
Hi @metmajer,
would you support using Github project release date: May 2020? How about a tagged intermediate release end-of-march?

Reply:
we do not expect a march release ...

Reply:
updated it to May matching the Github project definition.